### ISSUE TITLE
Hyperlink are not clickable when browser is only occupying half the screen. Chrome.

### ISSUE THREAD

**Issue by brewern5** [microsoft/vscode#202673]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: N/A - webpage issue.
- OS Version: Windows 11

Steps to Reproduce:

For reference I am using Google Chrome Version 120.0.6099.217. When I have half my screen occupied with my browser, utilizing the in-text hyperlinks is not possible, unless the browser takes up the full screen and is then resized to the original half screen.

When trying to click on the blue hyperlinks in half screen mode on the downloads page for VS itself, it appears to just be blue text, rather than a hyperlink.

**Comment by IllusionMH** [comment]
/needsMoreInfo

How this is related to VS Code editor itself?

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's missing some basic information or in some other way doesn't follow our [issue reporting guidelines](https://aka.ms/vscodeissuereporting). Please take the time to review these and update the issue.

Happy Coding!

**Comment by brewern5** [comment]
> /needsMoreInfo
> 
> How this is related to VS Code editor itself?

I should've made this more clear, this is for the downloads page itself on the VS page. 

What this has to do with VS? It is the page where you can download the IDE. Arguably this is one of the most important places for VS and VSC. There is no other place that is apparently obvious to report an issue, other than the glaring "report an issue" in the "In this article" sidebar on this aforementioned page (which does not state it is ONLY for VS). So if there is another place to report website issues rather than VS issues, please let me know. And maybe that should be more apparent on the site that the "Report Issues" link is only for VS.

**Comment by IllusionMH** [comment]
Now it's clearer that not some random behavior on unrelated website.

Do you mean this page https://code.visualstudio.com/Download or other page?
I wasn't able to reproduce behavior on this page, but I'm on Edge Dev

**Comment by brewern5** [comment]
I apologize, it is not the downloads page, I was on it yesterday and misspoke. 

Here is the link: https://code.visualstudio.com/docs/csharp/get-started 

I just tried to reproduce on Edge and it seems to work as intended. However, the issue still persists on Chrome (unsure about Firefox and other browsers). In fact it seems to be all buttons, hyperlinks, and dropdowns int the <main> section(referring to the <div role=main id="main-content"> in the HTML content). The footer and NAV sections seem to be unaffected. 

As for further testing, it is not just this page in particular. It is on ALL pages accessible from the NAV bar (Excluding download and Extensions.) However, when I inspected the page and looked a HTML/CSS. Everything worked as intended.
